scripts/uploadComponent.js

```diff
@@ -27,14 +27,29 @@ if (fs.existsSync(commander.crxFile)) {
   throw new Error('Missing or invalid crx file/directory', commander.crxFile, commander.crxDirectory)
 }
 
-util.createTableIfNotExists(commander.endpoint, commander.region).then(() => {
-  if (fs.lstatSync(crxParam).isDirectory()) {
-    fs.readdirSync(crxParam).forEach(file => {
-      if (path.parse(file).ext === '.crx') {
-        util.uploadCRXFile(commander.endpoint, commander.region, path.join(crxParam, file))
-      }
-    })
-  } else {
-    util.uploadCRXFile(commander.endpoint, commander.region, crxParam)
-  }
+let uploadJobs = []
+if (fs.lstatSync(crxParam).isDirectory()) {
+  fs.readdirSync(crxParam).forEach(file => {
+    if (path.parse(file).ext === '.crx') {
+      uploadJobs.push(util.uploadCRXFile(commander.endpoint, commander.region, path.join(crxParam, file)))
+    }
+  })
+} else {
+  uploadJobs.push(util.uploadCRXFile(commander.endpoint, commander.region, crxParam))
+}
+
+Promise.all(uploadJobs).then(() => {
+  util.createTableIfNotExists(commander.endpoint, commander.region).then(() => {
```

Comment:
I think this should happen inside of `updateDynamoDB`

Reply:
there may still be a race condition here for new components and also I think it's just cleaner that way

Reply:
@simonhong do we need to create a follow up issue to address this?

Reply:
IMO, current steps seems ok - Uploading all crx files -> create table -> update db for each crx files.
Isn't it? Or maybe I didn't understand well about create table step.